numpy/core/multiarray.py

```diff
@@ -251,6 +251,21 @@ def concatenate(arrays, axis=None, out=None, *, dtype=None, casting=None):
     return arrays
 
 
+def concat(arrays, axis=None, out=None, *, dtype=None, casting=None):
```

Comment:
```suggestion
def concat(arrays, axis=None, out=None, dtype=None, casting=None):
```
That asterisk doesn't need to be there.

Reply:
> Okay, the suggested changes should fix the PR. That being said, even if it's not necessary, you should still fix `runtests.py` for any future PRs.

Okay, Sir, I am fixing it as soon as possible. Thank you.